dist/src/main/java/io/camunda/zeebe/gateway/ActorSchedulerComponent.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway;
+
+import io.camunda.zeebe.gateway.impl.configuration.GatewayCfg;
+import io.camunda.zeebe.scheduler.ActorScheduler;
+import io.camunda.zeebe.shared.ActorClockConfiguration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+@Component
+final class ActorSchedulerComponent {
+
+  private final GatewayCfg cfg;
+  private final ActorClockConfiguration clockConfiguration;
+
+  @Autowired
+  public ActorSchedulerComponent(
+      final GatewayCfg cfg, final ActorClockConfiguration clockConfiguration) {
+    this.cfg = cfg;
+    this.clockConfiguration = clockConfiguration;
+  }
+
+  @Bean("actorScheduler")
+  @Autowired
```

Comment:
:wrench: Would recommend `@Scope("singleton")` as well. Can't imagine us needing more than one. Right now it's probably fine since we only autowire it in one place, but just in case.

Reply:
That's the default anyway, should we still add it even though we will probably never use anything different than "singleton"? :thinking:

Reply:
I've left it as is for now, without the default singleton scope

Reply:
That's fair. I'm not a fan of relying on implicit defaults, but we can revisit this as we add more beans.